misc/package/Data Files/MWSE/core/mcm/i18n/eng.lua

```diff
@@ -27,4 +27,28 @@ return {
 	["SET %s KEYBIND."] = "SET %%s KEYBIND.",
 	["SET NEW KEYBIND."] = "SET NEW KEYBIND.",
 	["Press any key to set the bind or ESC to cancel."] = "Press any key to set the bind or ESC to cancel.",
+	["Trace"] = "Trace",
+	["Debug"] = "Debug",
+	["Info"] = "Info",
+	["Warn"] = "Warn",
+	["Error"] = "Error",
+	["None"] = "None",
+	["Log options description"] = "\z
+	Change the current logging message severity level. A value of 'Warn' or 'Info' is recommended, \z
+	unless you're troubleshooting something. Each setting includes all the log messages of the previous setting. \z
+	Here is an explanation of the options:\n\n\t\z
+	\z
+	None: nothing will be logged.\n\n\t\z
+	\z
+	Error: error messages will be logged.\n\n\t\z
+	\z
+	Warn: error and warning messages will be logged.\n\n\t\z
+	\z
+	Info: error, warning and info messages will be logged.\n\n\t\z
+	\z
+	Debug: error, warning, info and debug messages will be logger. You may notice a decrease in performance.\n\n\t\z
```

Comment:
I think it should be "will be logged" instead of "will be logger"

Reply:
Done